conf/modules/nav_basic_rotorcraft.xml

```diff
@@ -11,7 +11,7 @@
       <dl_settings NAME="NAV">
         <dl_setting var="flight_altitude" MIN="0" STEP="0.1" MAX="400" module="navigation" unit="m" handler="SetFlightAltitude"/>
         <dl_setting var="nav_heading" MIN="0" STEP="1" MAX="360" unit="1/2^12r" alt_unit="deg" alt_unit_coef="0.0139882"/>
-        <dl_setting var="nav_radius" MIN="-50" STEP="0.1" MAX="50" unit="m"/>
```

Comment:
Thanks for the tip, will change. However, I think it would be nice to have an additional option to accurately set the value of a slider with the keyboard as well. Maybe a right click on the slider can open a dialog?

Reply:
It is possible to edit by hand a value with the new GCS, I will not update the old one.

Reply:
Ah I was trying that on the new GCS, just need to figure out how then.